llm/.env.example

```diff
@@ -0,0 +1,11 @@
+# WANDB
+WANDB_API_KEY=1234567890
```

Comment:
It might be worth mentioning in the readme that once logged in to wandb, you can get your API key by visiting https://wandb.ai/authorize so that people don't have to dig too much.

Reply:
Added.